server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/DatafileManager.java

```diff
@@ -571,4 +583,27 @@ public MetadataUpdateCount getUpdateCount() {
     return metadataUpdateCount.get();
   }
 
+  public void clearTransactions() {
+    tabletLog.clearLog();
+  }
+
+  public void resetTransactions() {
+    synchronized (tablet) {
+      tabletLog.resetLog(datafileSizes.keySet());
+    }
+  }
+
+  public void logTransactions() {
+    log.error("Operation log: {}", tabletLog.dumpLog());
+  }
+
+  public void checkTransactionLog() {
```

Comment:
Given this compares datafileSizes and tableLog should sync... or make the method private if its only called by code that is already synced

```suggestion
  public synchronized void checkTransactionLog() {
```

Reply:
I can make it private.